pom.xml

```diff
@@ -38,7 +38,7 @@
 
         <github.url>scm:git:git@github.com:dockstore/dockstore-support.git</github.url>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <dockstore-core.version>1.16.0-alpha.16</dockstore-core.version>
+        <dockstore-core.version>1.16.0-SNAPSHOT</dockstore-core.version>
```

Comment:
Probably should be versioned, I think you can use a dated snapshot in a pinch 
Like 
```
<dependency>
    <groupId>io.dockstore</groupId>
    <artifactId>bom-internal</artifactId>
    <version>1.16.0-20240926.155529-115</version>
    <type>pom</type>
</dependency>
```
but I haven't tried it yet

Reply:
Yes, I'll update this once I create a new webservice tag :slightly_smiling_face: